### PR TITLE
Update Readme with Setup instructions

### LINKED ISSUE #90: Update Readme with Setup instructions
## Acceptance Criteria

- [ ] Make Readme More attractive.
- [ ] Add Setup instructions

### PR DESCRIPTION
**Pull request by ninadvyas**
# Update Readme with Setup instructions : Closes #90 

## Acceptance Criteria fulfillment

- [x] Make Readme More attractive.
- [x] Add Setup instructions

## Proposed changes (screenshots)

<table>
  <tr>
    <td valign="top">
    <h2> Before </h2> 
    <a href="https://unwrapped.dev" style="flex: 1;">
        
    </td>
    <td valign="top">  
    <h2> After </h2> 
     <a href="https://unwrapped.dev" style="flex: 1;">
        
</a>
    </td>
  </tr>
</table>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,84 @@
...
   <h3>
     <i>
-      Spotify Unwrapped - for devs
+      Dive into Unwrapped - for coders
       <br/>
-      Showcase your contributions just like you flex your music!
+      Reveal your coding journey just as you showcase your musical taste!
       <br />
-      Check it out at https://unwrapped.dev
+      Explore more at  https://unwrapped.dev
```
Comment (jayantbh) [inline README.md:11]: I think this change isn't needed for now.
Happy to hear your reasoning behind why it helps, if it does.

**Comment by jayantbh** [comment]
@ninadvyas the before after screenshots aren't showing up for me
image


@samad-yar-khan might be able to confirm if that's the same

**Comment by samad-yar-khan** [comment]
> @ninadvyas the before after screenshots aren't showing up for me image
> 
> @samad-yar-khan might be able to confirm if that's the same for him

@jayantbh  they are showing for me

File: README.md [hunk 2]
```diff
@@ -1,72 +1,84 @@
...
+
+
+
+![Stargazers](https://img.shields.io/github/stars/middlewarehq/unwrapped?style=social) 
+![Forks](https://img.shields.io/github/forks/middlewarehq/unwrapped?style=social) 
+![Contributors](https://img.shields.io/github/contributors/middlewarehq/unwrapped?color=dark-green) 
+![Issues](https://img.shields.io/github/issues/middlewarehq/unwrapped) 
+![License](https://img.shields.io/github/license/middlewarehq/unwrapped) 
+
+
```
Comment (samad-yar-khan) [inline README.md:29]: @ninadvyas  thankss

File: README.md [hunk 3]
```diff
@@ -76,34 +88,35 @@ This app is designed to provide you with insights into your coding activities th
...
+# License
 
-## License
-This project is licensed under the MIT License - see the LICENSE.md file for details.
+This project is licensed under the [MIT License](https://github.com/middlewarehq/unwrapped/blob/main/LICENSE.md) - see the LICENSE.md file for details.
 
 
-<a href="https://unwrapped.dev"><img src="https://github.com/middlewarehq/unwrapped/assets/7949047/49024f49-86bc-49d5-acb6-3cc8037bface" width="100%" /></a>
+![Unwrapped-Image](https://github.com/devdnv/wrapped/assets/147969535/2cf9b09b-18d3-4b41-aeab-c547529c2ba5)
```
Comment (samad-yar-khan) [inline README.md:122]: @ninadvyas  Please use the older images

**Comment by ninadvyas** [comment]
@jayantbh of the changes didn't see on your side you can check out on  https://github.com/ninadvyas/unwrapped 
you can see the changes in readme file in my forked repository.

File: README.md [hunk 4]
```diff
@@ -1,72 +1,84 @@
...
+
+<h1 align='center'>🚀 Features 🚀</h1>
+
+- **Code Contributions 🎖️**: View a summary of your contributions across various repositories over the past year.
+- **Productivity Insights 👩‍💻**: Discover time spent waiting for reviews and identify top contributors to your code reviews.
+- **Repo Highlights**: Showcase noteworthy projects you've worked on during the year.
+- **Coding Habits 🦉**: Gain insights into your coding patterns, including peak coding hours and most active days.
+- **Contribution Styles 🌳**: Understand your contribution habits—are you more focused on pushing code or stuck in pull request reviews?
+- **Early Bug Detection 🚒**: Identify trends in bug detection and code changes before they reach production.
+- **And Much More 🧑‍🚀**
```
Comment (jayantbh) [inline README.md:45]: Might want to reduce the scale of the changes down a bit.
We definitely don't have anything to do with bug detection in this repo.

However, some of the languages changes are nice. Don't undo all your changes.

Some of the changes feel like changes made for the sake of increasing line count.

**Review by samad-yar-khan — DISMISSED** [review]
@ninadvyas  thanks for this contribution. The format changes look good but please use the older images. I have also mentioned where language changes are not needed.

File: README.md [hunk 5]
```diff
@@ -1,72 +1,84 @@
...
-- **Code Contributions :medal_military:**: View a summary of your contributions to various repositories over the past year. See where you stand !
-- **Productivity Insights :woman_technologist:**: Discover how long you spent waiting for reviews, and who helped you out the most with code reviews.
-- **Repo Highlights**: Highlight noteworthy projects you worked on during the year.
-- **Coding Habits :owl:**: Gain insights into your coding habits, such as peak coding hours and most active days.
-- **Cobtribtuion Styles :evergreen_tree:**: Are you pushing your code, or are you stuck in Pull Request Reviews ?
-- **Catching fires before they spread :firefighter:**: Are you often requesting changes and finding bugs before they reach production ?
-- **And Much More :woman_astronaut:**
-
-<a href="https://unwrapped.dev"><img src="https://github.com/middlewarehq/unwrapped/assets/7949047/390e9d7a-183e-41c6-bcde-599d91df732f" width="100%" /></a>
-
```
Comment (jayantbh) [inline README.md:33]: I think undo the image changes between features and project setup section.
Super appreciate you taking a stab at it.

Here's the reasons behind why it was the way it was:
1. Limit the number of images so it doesn't need too much scrolling to get to the main thing.
2. The height of the existing banner was chosen specifically so a few more names of the popular devs show up in it. The reduced height further crops those out.

**Comment by ninadvyas** [comment]
ok. Thanks for the feedback ill let you know after I fixed the changes @samad-yar-khan

**Comment by ninadvyas** [comment]
Do you need anything else to be added @samad-yar-khan rather then the changes that you have mentioned??

File: README.md [hunk 6]
```diff
@@ -76,34 +88,35 @@ This app is designed to provide you with insights into your coding activities th
...
+     ```
+- #### Third-Party Services
+  - #### [Mixpanel](https://mixpanel.com) for usage metrics:
+   ```bash
+      NEXT_PUBLIC_MIXPANEL=NEXT_PUBLIC_MIXPANEL
+     ```
+  - #### [Sentry](https://sentry.io/welcome/)  for issue tracking::
+   ```bash
+      NEXT_PUBLIC_MIXPANEL=NEXT_PUBLIC_MIXPANEL
+     ```
```
Comment (jayantbh) [inline README.md:115]: I'd say undo changes in this entire section.

The nested lists aren't needed here. They add indentation that doesn't make it look that much better either.

Making the service names bolder is nice.

No need to specify AWS as optional. This whole section is already optional.

File: README.md (hunk 3 above)
Comment (jayantbh) [inline README.md:122]: This graphic doesn't adhere to our design schematic.
Unfortunately we don't have it written down anywhere for me to share with you, but hopefully in the future.

For now, please remove this one.

File: README.md [hunk 7]
```diff
@@ -1,72 +1,84 @@
...
+  curl -o- -L https://yarnpkg.com/install.sh | bash
+  export PATH="$HOME/.yarn/bin:$PATH"
+  ```
+
+- #### Node.js Installation (using Node Version Manager - nvm):
+   ```bash
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+    nvm install 20.10.0
+    nvm use 20.10.0
+    ```
```
Comment (jayantbh) [inline README.md:64]: This seems to have unneeded indentation.

**Review by jayantbh — CHANGES_REQUESTED** [review]

**Review by jayantbh — APPROVED** [review]

**Comment by samad-yar-khan** [comment]
Thanks for the contribution @ninadvyas 🚀

**Pull request merged**